shesha-reactjs/src/components/reactTable/index.tsx

```diff
@@ -378,24 +388,53 @@ export const ReactTable: FC<IReactTableProps> = ({
       <div className={mainStyles.shaReactTable} style={containerStyleFinal}>
         <div {...getTableProps()} className={styles.shaTable} style={tableStyle}>
           {columns?.length > 0 &&
-            headerGroups.map(headerGroup => {
+            headerGroups.map((headerGroup) => {
               const { key, ...headerGroupProps } = headerGroup.getHeaderGroupProps();
               return (
                 <div
                   key={key}
                   {...headerGroupProps}
                   className={classNames(styles.tr, styles.trHead)}
+                  style={{ ...fixedHeadersStyle, display: 'flex' }} // Make the header row sticky
                 >
-                  {headerGroup?.headers?.map((column) => {
-                    const { key, ...headerProps } = column.getHeaderProps(column.getSortByToggleProps());
+                  {headerGroup?.headers?.map((column, index) => {
+                    const isFixed = (column as any)?.isFixed;
```

Comment:
@nthangeniphumudzo avoid cast to `any`, we should use types/interfaces, otherwise your code may be broken later